script/generatecert

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+set -x
+
+mkdir cert
```

Comment:
There's no real need to create a `cert` directory, just store them in the current directory.